File: Iniciante/URI 1017.cs
using System;

class URI {
 
    static void Main(string[] args) { 
        int tempo = int.Parse(Console.ReadLine());
        int velocidade = int.Parse(Console.ReadLine());
        double kilometros = tempo * velocidade;
		double media = kilometros / 12;
        Console.WriteLine("{0:F3}", media);
    }
    
}

File: Iniciante/URI 1017.dart
import 'dart:io';

void main() {
  double tempo = double.parse(stdin.readLineSync());
  double velocidade = double.parse(stdin.readLineSync());
  double kilometros = tempo * velocidade;
  double media = kilometros / 12;
  print(media.toStringAsFixed(3));
}
